### PR TITLE
Linkify headers and a few references, add twilio example

### PR DESCRIPTION
**Pull request by ArtskydJ**
- Linkify headers and remove "See full documentation" links
- Add links to a few references to some just-login modules
- Add twilio example

### CODE REVIEW

File: build/info.md [hunk 1]
```diff
@@ -59,24 +59,46 @@ core.authenticate(token, function (err, addr) {
 
 ## Not using Email
 
-This is the point where you must send the token to the user somehow. Here is an example of using twitter's direct messaging instead of email!
+You must send the token to the user somehow. One way is to send them a link with the token in it, and authenticate tokens sent to that endpoint.
```
Comment (ArtskydJ) [inline build/info.md]: IDK if I like how this is worded.....

Comment (TehShrike) [inline build/info.md]: > One way is to send them a link with the token in it, and have the server authenticate the token when that url is visited.

mebbe?

Comment (ArtskydJ) [inline build/info.md]: :+1:

**Comment by TehShrike** [comment]
lgtm

**Pull request merged**